docs/source/cmd_validate.rst

```diff
@@ -32,18 +32,19 @@ validate a NeXus file
 
 .. rubric:: command line help
 
-.. code-block:: console
-   :linenos:
+..  code-block:: console
+    :linenos:
 
-   usage: punx validate [-h] [--report REPORT] infile
+    usage: punx validate [-h] [--report REPORT] infile
 
-   positional arguments:
-     infile           HDF5 or NXDL file name
+    positional arguments:
+      infile           HDF5 or NXDL file name
```

Comment:
Was this added whitespace intentional?

Reply:
Intentional.  VSCode indents things on 4-columns and this one was on 3.  A few other places will see this as well.